### PR TITLE
fix(perf): add call to performance.flush

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by ampsauce** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 8ab2ce3afb919e714c4fe00735c63278aad4560f
Build details: https://travis-ci.org/ampsauce/amphtml/builds/93446125

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: src/amp.js [hunk 1]
```diff
@@ -66,6 +66,8 @@ try {
     } finally {
       makeBodyVisible(document);
       perf.tick('e_is');
+      // NOTE(erwinm): move flush when have the new ticks in place.
```
Comment (dvoytenko) [inline src/amp.js]: Please clarify the sentence. Not sure if this is "move *to* flush" or such? Also, we don't do "NOTE". It's either a TODO or not :)

Comment (erwinmombay) [inline src/amp.js]: done.

**Comment by ampsauce** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: f301c8c4d802112150b40434061ce60e30631ab1
Build details: https://travis-ci.org/ampsauce/amphtml/builds/94059704

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by dvoytenko** [comment]
LGTM. I Merged it accidentally. Hope it's ok.

**Pull request merged**